### ISSUE TITLE
Active Claims detail | Mobile alert

### ISSUE THREAD

**Issue by bessgreen** [department-of-veterans-affairs/va.gov-team#17322]
Note: Patrick is implementing. Saving story for FE team in the backlog in the event that we need to reference a11y criteria.

## User Story
As a user, I want to know if the VA needs me to take action / provide information for my active claim.  

## Acceptance Criteria
1. On the Active Claims detail screen, pull in the Mobile Alert common component
2. This should only display if an alert comes through in the API
     - Hide this from the UI if there is no alert

## Designs
https://adhoc.invisionapp.com/d/main/#/console/20546460/432345660/inspect
![Screen Shot 2020-12-10 at 5.45.55 PM.png](https://images.zenhubusercontent.com/5f64e41291069a617acc1b47/0bc2b1b7-8d72-46f1-ae2a-3f829658794c)

## Accessibility Acceptance Criteria
1. Everything on the page should be read by the screen reader.
     - If anything is being conveyed visually only (buttons, menu items, list, tables, images, headings, forms, etc) the component type needs to be announced by the screenreader. (Include name (title), role, value)
2. Each page should support text resizing based on the OS font scaling settings without loss of content, content overlap, or horizontal scroll.
3. Each touch target must be at minimum 44px by 44px
4. The app needs to support operability by tap (no gestures) or voice control only. In cases where gestures occur, e.g. scroll, ensure these simple gestures are operable through the OS’ built-in capabilities.  
5. All elements on a page should be able to be traversed by an auxiliary keyboard. As a user uses the keyboard to navigate a page, its focus element should be clear.
6. When a page opens, the first section or element should automatically be placed in focus
7. When a page opens, the title page should be announced

## Accessibility Implementation Guidelines 
**Mobile Alert**
- Treat as static text

## Dev Checklist
- [x] UX/UI
- [ ] Accessibility
- [x] Add tests

**Comment by msmanak** [comment]
I see that the segment control button won't pass the 44px height during testing so follow our orginal instruction for design height.

**Comment by lexicalninja** [comment]
its fixed in a different ticket/PR
department-of-veterans-affairs/va.gov-team#17362
department-of-veterans-affairs/va.gov-team#16459
department-of-veterans-affairs/va-mobile-app/pull/302

These should take care of that throughout the app

**Comment by digitalken** [comment]
Added to TestRail.

**Comment by digitalken** [comment]
PASS 12/18/2020. Moving to design QA.
